GenderPayGap.WebUI/Views/AccountCreation/CreateUserAccount.cshtml

```diff
@@ -0,0 +1,157 @@
+﻿@using GenderPayGap.WebUI.Models.AccountCreation
+@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@model GenderPayGap.WebUI.Models.AccountCreation.CreateUserAccountViewModel
+@{
+    ViewBag.Title = "Create my account - Gender pay gap reporting service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
+
+@section BeforeMain {
+    @{ await Html.RenderPartialAsync("/Views/ReportingStepByStep/ReportingPageHeader.cshtml"); }
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">Create my account</h1>
+    </div>
+</div>
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <p class="govuk-body">
+            You will need to create your account, and then add your organisation(s) to this account.
+        </p>
+        <p class="govuk-body">
+            Once your email address is confirmed you will then be able to report your gender pay gap data.
+        </p>
+        <p class="govuk-body">
+            If you already have a user account, please
+            @Html.ActionLink("sign in", nameof(OrganisationController.ManageOrganisations),
+                "Organisation").
+        </p>
+        <div class="govuk-inset-text">
+            You will need to create your own user account even if someone else has previously reported
+            your organisation's gender pay gap.
+        </div>
+
+        <div class="govuk-grid-column-two-thirds">
+            <form method="POST" action="@Url.Action("CreateUserAccountPost", "AccountCreation")">
+                @Html.AntiForgeryToken()
+                <div class="govuk-form-group">
+
+                    @(Html.GovUkErrorSummary(new[]
+                    {
+                        nameof(CreateUserAccountViewModel.EmailAddress),
+                        nameof(CreateUserAccountViewModel.ConfirmEmailAddress)
```

Comment:
Is there a reason why you've specified 2 fields here?
I'd normally expect the see all the fields listed, or none of them.

Reply:
I've added them all in now, thanks